aggregator.go

```diff
@@ -41,6 +43,11 @@ const (
 	ExitStatusServerError
 )
 
+var (
+	serverInstance   *server.HTTPServer = nil
```

Comment:
probably linters wont be happy with "= nil" part as it is redundant

Reply:
fixed